### PR TITLE
Refactor k8s custom values file

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-7871

# Done
- Delete custom config files.
- Fix to use custom config files of `scalar-kubernetes` repo.
- Some refactoring

### CODE REVIEW

**Comment by tei-k** [comment]
Sorry! I'm going to split this PR.

**Comment by tei-k** [comment]
Removed the part of `helm delete`, So this PR is just only for refactoring.

File: test/src/integration/integration_test.go [hunk 1]
```diff
@@ -114,6 +114,11 @@ func TestEndToEndK8s(t *testing.T) {
 	test_structure.RunTestStage(t, "validate", func() {
 		t.Run("TestScalarDL", TestScalarDL)
 	})
+
+	test_structure.RunTestStage(t, "delete", func() {
+		logger.Logf(t, "Run helm delete")
+		runHelmDelete(t)
```
[the remark this answers is not in the document]
Comment (tei-k) [reply test/src/integration/integration_test.go]: OK! I'll try it in another PR.

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Review by ymorimo — APPROVED** [review]
LGTM! Good refactoring.

**Pull request merged**